airflow/jobs/triggerer_job_runner.py

```diff
@@ -530,11 +530,9 @@ async def create_triggers(self):
         while self.to_create:
             trigger_id, trigger_instance = self.to_create.popleft()
             if trigger_id not in self.triggers:
-                ti: TaskInstance = trigger_instance.task_instance
                 self.triggers[trigger_id] = {
                     "task": asyncio.create_task(self.run_trigger(trigger_id, trigger_instance)),
-                    "name": f"{ti.dag_id}/{ti.run_id}/{ti.task_id}/{ti.map_index}/{ti.try_number} "
```

Comment:
To simplify it I removed the DAG related information for the name. Because, with this change, a trigger might not be associated to a DAG. Let me know if this is a concern. @dstandish?

Reply:
Yes I was not sure either. To be honest I picked the trigger ID for simplicity because I was not sure how much this `name` was used. But I agree, we can do better here. I'll update it. Thanks!

Reply:
yeah even i dont have much information to share better name pattern here :)

Reply:
I set back the name if a task instance is associated to the trigger. However, if the trigger is used for driven event scheduling, I use the trigger ID as name since this is the only information I have handy

Reply:
cool :)